### PR TITLE
DOCS-5226 GitHub Actions Edits

### PR DESCRIPTION
**Pull request by alai97**
## Context

Adds a site region shortcode in the GitHub Actions doc page.

## How?

DOCS-5226

## Preview

https://docs-staging.datadoghq.com/alai97/continuous-testing-cicd-integrations-site-region-update/continuous_testing/cicd_integrations/github_actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ jobs:
...
 | `subdomain`         | string  | _optional_  | The name of the custom subdomain set to access your Datadog application. If the URL used to access Datadog is `myorg.datadoghq.com`, the subdomain value needs to be set to `myorg`. **Default:** `app`. |
 | `files`             | string  | _optional_  | Glob pattern to detect Synthetic tests config files. **Default:** `{,!(node_modules)/**/}*.synthetics.json`.                                                                                             |
-| `datadog_site`      | string  | _optional_  | The Datadog site. For users in the EU, set to `datadoghq.eu`. For example: `datadoghq.com` or `datadoghq.eu`. **Default:** `datadoghq.com`.                                                              |
+| `datadog_site`      | string  | _optional_  | The [Datadog site][11]. Your Datadog site is {{< region-param key="dd_site" code="true" >}}. **Default:** `datadoghq.com`.                                                              |
```
Comment (etnbrd) [inline README.md:120]: As per the [documentation about region shortcode](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1183023503/Regions+Sites#Site-Region-Shortcode):
> Note: This shortcode only works in the documentation and dogweb repo. For integrations-core or integrations-extras, use the site region partial.

```suggestion
| `datadog_site`      | string  | _optional_  | The [Datadog site][11]. <!-- partial Your Datadog site is {{< region-param key="dd_site" code="true" >}}. partial --> **Default:** `datadoghq.com`.                                                              |
```

**Review by etnbrd — CHANGES_REQUESTED** [review]
Thanks a lot for the improvements 🙏 
Unfortunately, I think it'd be preferable for the github rendering of this markdown to not leak the shortcodes that are used for the internal documentation.
Screenshot 2023-04-13 at 10 12 51 AM

**Review by etnbrd — APPROVED** [review]
LGTM 👍 
Thanks for taking this review into account 🙏

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**